src/test/ui/__check/issue-63695.rs

```diff
@@ -0,0 +1,14 @@
+// run-pass
```

Comment:
The directory this test is in seems odd; shouldn't it be in src/test/ui/const-generics?

Reply:
will move it to const-generics one this test actually runs, this makes it simpler to only test
these tests while developing